README.md

```diff
@@ -833,6 +834,67 @@ Response:
 }
 ```
 
+## Delete
+To delete data from the vault, use the `delete(records, options?)` method of the Skyflow client. The `records` parameter takes an array of records to delete in the following format. The `options` parameter is optional and takes an object of deletion parameters. Currently, there are no supported deletion parameters.
+
+Call schema:
+
+```java
+JSONObject records = new JSONObject();
+JSONArray recordsArray = new JSONArray();
+
+JSONObject record = new JSONObject();
+
+record.put("id", "<SKYFLOW_ID_1>");
+record.put("table", "<TABLE_NAME>");
+recordsArray.add(record);
+records.put("records", recordsArray);
+
+DeleteOptions options = new DeleteOptions;
+skyflowClient.delete(records, options);
+```
+
+An example of delete call:
+```java
+JSONObject records = new JSONObject();
+JSONArray recordsArray = new JSONArray();
+
+JSONObject record = new JSONObject();
+record.put("id", "71be4592-b9af-4dec-8669-5b9c926afb4c");
+record.put("table", "cards");
+recordsArray.add(record);
+        
+JSONObject record2 = new JSONObject();
+record2.put("id", "2adf32e7-9a04-408e-b8bb-5b0a852422e0");
+record2.put("table", "cards");
+recordsArray.add(record2);
+
+records.put("records", recordsArray);
+DeleteOptions options = new DeleteOptions;
```

Comment:
As options are optional,  need not to mention.

Reply:
changes done